barretenberg/cpp/src/barretenberg/vm2/simulation_helper.cpp

```diff
@@ -39,6 +40,7 @@ struct FastSettings {
     using AluEventEmitter = NoopEventEmitter<AluEvent>;
     using MemoryEventEmitter = NoopEventEmitter<MemoryEvent>;
     using AddressingEventEmitter = NoopEventEmitter<AddressingEvent>;
+    using BytecodeRetrievalEventEmitter = NoopEventEmitter<BytecodeRetrievalEvent>;
     using BytecodeHashingEventEmitter = NoopEventEmitter<BytecodeHashingEvent>;
     using BytecodeDecompositionEventEmitter = NoopEventEmitter<BytecodeDecompositionEvent>;
     // Customization can go here, for example a BytecodeManager that does NOT hash bytecodes.
```

Comment:
Forgot to remove this line, will do in follow up PR unless I have to change sth here.